static/bidder-info/adelement.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "https://amp.adelement.com/openrtb2/auction?supply_id={{.SupplyId}}"
+endpointCompression: gzip
+maintainer:
+  email: "tech@adelement.com"
+gvlVendorID: 16
```

Comment:
This gvl vendor ID belongs to RTBHouse. Could you please confirm if this is correct?

Reply:
I have updated it to Adelement vendor Id 196.